API.md

```diff
@@ -1205,3 +1205,5 @@ Returns **([Array][68]\<Warning> | [Promise][64]<[Array][68]\<Warning>>)** An ar
 [75]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
 
 [76]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function
+
+[77]: https://developer.mozilla.org/en-US/docs/Web/API/Document
```

Comment:
Was this change generated by `yarn run docs`?  If not, we likely need to investigate why `yarn run docs` isn't producing this change, as `API.md` is an artifact of that build step

Reply:
I was asked to update the docs before merging this Definitely Typed PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55123

I've never contributed to this repo before and I made these changes manually. I tried running `yarn run docs` and it removed all of my changes. What do I need to do to make this change mergeable?